### DISCUSSION TITLE
如何从现有的旧版本的非本项目发行的armbian升级到本项目发行的新版本的armbian？

### DISCUSSION THREAD

**Discussion by BrandonStudio** [ophub/amlogic-s9xxx-armbian#2166] [Amlogic]
之前TF卡刷了一个buster版本的armbian，但是没有`armbian-install`命令。怎么能从现有系统升级，同时保留数据？

**Comment by ophub** [comment]
armbian系统可以一路升级到最新版，系统自带的版本升级功能即可。
这个仓库的armbian和官方是一致的，只是在官方的基础上扩增了几个命令，例如armbian-software等，并没有改变官方的使用方法。我对官方的修改有2点，一个是增加了几个便捷的命令，另外一个是扩展了更多电视盒子的引导启动（仅修改了/boot的引导启动方法，对系统的使用无影响）。

所以你按照官方的方法可以一直升级到最新的jammy，在/usr/sbin里添加这个仓库的armbian-sync，并添加/etc/ophub-release这2个文件，就可以使用armbian-sync命令一键拉去到全部的独立添加的脚本等。
